netket/variational/base.py

```diff
@@ -219,16 +219,21 @@ def quantum_geometric_tensor(self, qgt_T):
         """
         raise NotImplementedError
 
-    def to_array(self, normalize: bool = True) -> jnp.ndarray:
+    def to_array(
+        self, normalize: bool = True, stable: bool = True, exp: bool = True
+    ) -> jnp.ndarray:
         """
         Returns the dense-vector representation of this state.
 
         Args:
             normalize: If True, the vector is normalized to have L2-norm 1.
+            stable: If True, internally subtract the maximal entry of the vector in log space
+              to enhance numerical stability. If also `normalize = False`, the results will be
+              different from those of `model.apply`.
+            exp: If True, the vector is in exponential space.
```

Comment:
I think this is confusing, it would make more sense to call it ```log``` and it is True (False by default) it will return the log of the vector instead of the actual vector

Reply:
Ok, I'll change it. Actually this argument is not used for now. I tried several ways to compute that sum of probabilities, but there was no improvement in accuracy.